CHANGELOG.md

```diff
@@ -19,6 +19,10 @@ nav_order: 1
 - Use the code-generated client to manage `aiven_service_integration` and `aiven_service_integration_endpoint`
 - Use Go 1.23
 
+## [4.25.0] - 2024-09-13
+
+- Add capability to set description and owner group per topic.
+
```

Comment:
Please just add the change to the upper list. We add the headline when we are ready to release it :)